app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt

```diff
@@ -84,6 +86,9 @@ class EventsFragment : Fragment(), BottomIconDoubleClick {
             .nonNull()
             .observe(this, Observer { list ->
                 eventsListAdapter.submitList(list)
+                list.forEach {
+                    Timber.e(it.ticketPrices.toString())
```

Comment:
I'm getting app prices to 0.0 - 
```
[TicketPrice(id=820, price=0.0), TicketPrice(id=821, price=0.0), TicketPrice(id=822, price=0.0), TicketPrice(id=823, price=0.0), TicketPrice(id=824, price=0.0), TicketPrice(id=830, price=0.0), TicketPrice(id=819, price=0.0)]
```

Reply:
On chrome dev tools - 
![image](https://user-images.githubusercontent.com/27783786/60677931-8122ac80-9ea0-11e9-8fe3-080ce9f212c9.png)

Reply:
@iamareebjamal Is relationship work like that? It is giving correct if but the price to zero.

Reply:
You have just requested id so why will you get prices.

Besides, do you realize how bad this solution is? You'll be loading all tickets for every event this way

Reply:
> You'll be loading all tickets for every event this way

But it is loading id and price only for tickets which are required in event details fragment. And almost all variables in event details fragment are loading from the database.

Reply:
> You have just requested id so why will you get prices.

How I have added `?include=tickets`

Reply:
Give me the event ID

Reply:
> Give me the event ID

It is for all events. I have just pasted one.

Reply:
https://api.eventyay.com/v1/events/833?include=tickets

Reply:
Can I do one thing, Just fetch all tickets of events, insert them into the database and use same in tickets fragment? Also, revalidate tickets in the ticket fragment.

Reply:
Yes, you can do that but only in event detail fragment

Reply:
Also, separate the event detail query from ticket fetching query as it'll take a long time to query event with all the tickets

Reply:
Updated